packages/components/layout/src/Layout.styles.ts

```diff
@@ -12,16 +12,19 @@ export const getLayoutMaxWidthStyles = (variant: LayoutProps['variant']) => {
 
   return css({
     width: '100%',
-    maxWidth: '1550px',
-    padding: `0 ${tokens.spacingXs}`,
+    maxWidth: '1524px',
   });
 };
 
 export const getLayoutStyles = (variant: LayoutProps['variant']) => ({
-  root: css({
-    width: '100%',
+  root: css(getLayoutMaxWidthStyles(variant), {
+    backgroundColor: tokens.colorWhite,
+    boxShadow:
+      '0px 6px 16px -2px rgba(25, 37, 50, 0.1), 0px 3px 6px -3px rgba(25, 37, 50, 0.15)',
```

Comment:
@damann Is this a new token we want to reuse?

Reply:
The border-radius also does not match an existing token.